packages/shell-api/src/helpers.ts

```diff
@@ -614,18 +614,30 @@ export async function getPrintableShardStatus(
               );
             }
 
-            const tagsRes: any[] = [];
-            for await (const tag of (
+            const tags = await (
```

Comment:
It seems like, based on the ticket, the legacy shell printed at least some of the tags even if there were more than the given threshold, and then added the "too many tags" message after that. Should we do the same? That would also allow us to keep the `for await` instead of having to load the entire tag list into memory with `.toArray()`

Reply:
The comment and this behavior reflect the behavior we already have for chunks, that happen above in the same method. I assumed this was the intended behavior for the new shell. Is it really helpful to print some tags id we hide the rest anyway? How big is the chance that they want to see the first 5 out of 100..

Reply:
Hm yeah ... I'd still think it results in a better UX (for both cases, tbh) if we print a few and then tell the user that there's more – that gives them an impression of what these tag ranges look like, if nothing else.

Is there any reason not to do that?

Reply:
No reason. Should we also update chunks then?

Reply:
I think that would be an UX improvement, yes 🙂